hca-to-scea/helpers/multitab_excel_to_single_txt.py

```diff
@@ -46,57 +72,34 @@ def multitab_excel_to_dict(work_dir, excel_file):
 
     return xlsx_dict_clean
 
-
-def get_specimen(xlsx_dict: {}, cell_line_id: str) -> str:
-
-    """If a cell line was derived from a specimen_from_organism,
-    get the id of the specimen_from_organism from which it was derived.
-    """
-    specimen_from_organism_id = xlsx_dict['cell_line'].loc[
-            xlsx_dict['cell_line']['cell_line.biomaterial_core.biomaterial_id'] == cell_line_id][
-            'specimen_from_organism.biomaterial_core.biomaterial_id'].values[0]
-
-    return specimen_from_organism_id
-
-
-def replace_specimen_id_with_cell_line_id(merged_df: pd.DataFrame()) -> pd.DataFrame():
-
-    merged_df['specimen_from_organism.biomaterial_core.biomaterial_id'] = merged_df[
-        'specimen_from_organism.biomaterial_core.biomaterial_id'].fillna(
-        merged_df.loc[merged_df['specimen_from_organism.biomaterial_core.biomaterial_id'].isna()][
-            'cell_line.biomaterial_core.biomaterial_id'].apply(get_specimen))
-
-    return merged_df
-
-def merge_dataframes(xlsx_dict: {}) -> pd.DataFrame():
-
-    """Merge the sequence file df with the cell suspension df via the linked
-     cell suspension ids."""
+def merge_sample_types(xlsx_dict: {},experimental_design) -> pd.DataFrame():
 
     '''process.insdc_experiment.insdc_experiment_accession' is present in multiple tabs. Save the cell_suspension experiment accessions with a unique name.'''
-    xlsx_dict['cell_suspension']["cell_suspension.insdc_experiment.insdc_experiment_accession"] = list(xlsx_dict['cell_suspension']['process.insdc_experiment.insdc_experiment_accession'])
+    if 'process.insdc_experiment.insdc_experiment_accession' in xlsx_dict['cell_suspension'].columns:
```

Comment:
`xlsx_dict['cell_suspension']` repeats several times. Consider storing it in a variable.